sdk-reference/web/nextjs.mdx

```diff
@@ -69,146 +75,654 @@ export const config = {
     // Required for Rownd token handling
     '/api/rownd-token-callback',
     // Add your protected routes
-    '/protected/:path*'
+    '/protected/:path*',
+    '/api/protected/:path*'
   ]
 };
-
 ```
 
-## Authentication Components
+## Server-Side Features
 
-### Protected Routes / Pages
-To protect a page from being accessed by unauthenticated users, you can use
-the `withRowndRequireSignIn` higher-order component.
+The `@rownd/next/server` package provides server-side authentication utilities:
 
-```jsx
+```typescript
 import {
   withRowndRequireSignIn,
   getRowndUser,
   getAccessToken,
   isAuthenticated,
+  getRowndUserId,
+  ROWND_TOKEN_CALLBACK_PATH
 } from '@rownd/next/server';
 import { cookies } from 'next/headers';
+```
 
-async function ProtectedPage() {
+### Server-Side Functions
+
+| Function | Description | Parameters | Return Type | Usage Example |
+|----------|-------------|------------|-------------|---------------|
+| `getRowndUser` | Retrieves the current authenticated user's data from Rownd | `cookies: ReadonlyRequestCookies` | `Promise<RowndServerUser \| null>` | `const user = await getRowndUser(cookies)` |
+| `getRowndUserId` | Gets the current user's ID from server context | `cookies: ReadonlyRequestCookies` | `Promise<string \| null>` | `const userId = await getRowndUserId(cookies)` |
+| `getRowndAccessToken` | Retrieves the current access token from server context | `cookies: ReadonlyRequestCookies` | `Promise<string \| null>` | `const token = await getRowndAccessToken(cookies)` |
+| `isAuthenticated` | Checks if there is an authenticated user in server context | `cookies: ReadonlyRequestCookies` | `Promise<boolean>` | `const isAuth = await isAuthenticated(cookies)` |
+| `withRowndMiddleware` | Higher-order function for Next.js middleware to handle authentication | `middleware: NextMiddleware` | `NextMiddleware` | See middleware example below |
+| `withRowndRequireSignIn` | Higher-order function to protect routes/pages/API handlers | `handler: Function, cookies: ReadonlyRequestCookies` | `Function` | See protected routes example below |
+
+### Constants
+
+| Constant | Description | Value | Usage |
+|----------|-------------|--------|-------|
+| `ROWND_TOKEN_CALLBACK_PATH` | Path for handling Rownd token callbacks | `/api/rownd-token-callback` | Used in middleware matcher config |
+
+### Example Usage
+
+```typescript
+// Middleware setup
+export const config = {
+  matcher: [
+    ROWND_TOKEN_CALLBACK_PATH,  // Required for token handling
```

Comment:
This can't be a variable. It has to be the string literal `/api/rownd-token-callback`